SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -278,6 +282,15 @@ void USpatialActorChannel::DeleteEntityIfAuthoritative()
 			UE_LOG(LogSpatialActorChannel, Error, TEXT("DeleteEntityIfAuthoritative called on actor channel with null actor - entity id (%lld)"), EntityId);
 		}
 	}
+	else if(bCreatedEntity) // We will gain authority soon
+	{
+		if (Actor->GetTearOff())
+		{
+			// Send tear off update now
+			Sender->SendTearOffUpdate(Actor);
```

Comment:
I don't think we can do this now, as we don't have spatial authority to send this update. I'm not sure on the specifics here - @samiwh ?

Reply:
I don't think we can. We need to queue this update until we get authority.
N.B. In a test this might end up working sometimes because you gain authority before the worker knows it has authority.  It's quite likely that if you send the create request then send an update that you could have authority before the updates arrives. Even though the op hasn't arrived yet.